index.vue

```diff
@@ -72,12 +70,16 @@ export default
 					@scriptCodes.push script.replace /<\/?script[^>]*>/gi, ''
 
 		# add script to head
-		loadScript: (scriptURL) -> new Promise (resolve, reject) ->
-			script = document.createElement('script')
-			script.src = scriptURL
-			script.async = true
-			script.onload = resolve
-			script.onerror= reject
-			document.head.appendChild(script)
+		loadScript: (scriptURL) ->
+			console.log "Loading: " + scriptURL
+			# Capture html from `this`
+			html = @$refs.html
```

Comment:
What if you change this to:

```
target = @$refs.html || document.head
```

Sometimes I've found that a ref is empty.... not sure why ... this would give us a sane fallback for those times.